app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -96,6 +102,16 @@ public class CommCareSetupActivity extends CommCareActivity<CommCareSetupActivit
     private static final String KEY_SHOW_NOTIFICATIONS_BUTTON = "show-notifications-button";
     public static final int MAX_ALLOWED_APPS = 4;
 
+    private BiometricManager biometricManager;

```

Comment:
This code should be encapsulated elsewhere so it isn't directly in this activity, more thoughts in comments below